### ISSUE TITLE
is it possible to import contacts from gmail on ios?

### ISSUE THREAD

**Issue by AFMW** [joshuapinter/react-native-unified-contacts#34]
Is it possible with this library to import the contacts that i have on mi gmail account on IOS?

Im using RN 0.42.0

**Comment by joshuapinter** [comment]
Good question. I know Unified Contacts doesn't connect to your Gmail account (yet!) but I *think* iOS might have a way to synchronize your contacts with your Gmail account, and then from there you can access them via Unified Contacts.

If not, I'd be happy to accept a Pull Request to do this.

Let me know.

**Comment by AFMW** [comment]
I think it would be awesome that the library itself synchronize your Gmail contacts, It would save a lot of headaches to the ones who would use the PR app.

**Comment by joshuapinter** [comment]
@AFMW I agree. I'm going to close this for now but will add that to the development backlog.
